shoulder_pan_controller:
    type: dynamixel_hardware_interface/JointPositionController
    joint: shoulder_pan_joint
    max_velocity: 2.0
    motors:
        - id: 0
          init: 452
          min: 140
          max: 752

shoulder_pitch_controller:
    type: dynamixel_hardware_interface/JointPositionController
    joint: shoulder_pitch_joint
    max_velocity: 2.0
    motors:
        - id: 1
          init: 2040
          min: 861
          max: 2750

elbow_roll_controller:
    type: dynamixel_hardware_interface/JointPositionController
    joint: elbow_roll_joint
    max_velocity: 2.0
    motors:
        - id: 2
          init: 513
          min: 200
          max: 832

elbow_pitch_controller:
    type: dynamixel_hardware_interface/JointPositionController
    joint: elbow_pitch_joint
    max_velocity: 2.0
    motors:
        - id: 3
          init: 508
          min: 137
          max: 810

wrist_roll_controller:
    type: dynamixel_hardware_interface/JointPositionController
    joint: wrist_roll_joint
    max_velocity: 2.0
    motors:
        - id: 4
          init: 495
          min: 79
          max: 911

wrist_pitch_controller:
    type: dynamixel_hardware_interface/JointPositionController 
    joint: wrist_pitch_joint
    max_velocity: 2.0
    motors:
        - id: 5
          init: 507
          min: 135
          max: 863

gripper_roll_controller:
    type: dynamixel_hardware_interface/JointPositionController
    joint: gripper_roll_joint
    max_velocity: 2.0
    motors:
        - id: 6
          init: 522
          min: 74
          max: 971

gripper_finger_controller:
    type: dynamixel_hardware_interface/JointPositionController
    joint: gripper_finger_joint
    max_velocity: 2.0
    motors:
        - id: 7
          init: 281
          min: 103
          max: 460
